addons/auth_signup/models/res_users.py

```diff
@@ -149,8 +149,7 @@ def create(self, values):
     @api.multi
     def copy(self, default=None):
         self.ensure_one()
-        context = self.env.context
         if not default or not default.get('email'):
             # avoid sending email to the user we are duplicating
-            context['reset_password'] = False
-        return super(ResUsers, self.with_context(context)).copy(default=default)
+            return super(ResUsers, self.with_context(dict(self.env.context, reset_password=False))).copy(default=default)
```

Comment:
Just use `.with_context(reset_password=False)` rather than manually duplicating the whole context and resetting it.

And maybe alias the super-proxy to avoid having two different `copy` callsites e.g.
```python
def copy(self, default=None):
    self.ensure_one()
    sup = super(ResUsers, self)
    if not default or not default.get('email'):
        sup = super(ResUsers, self.with_default(reset_password=False)
    return sup.copy(default=default)
```